src/orquestra/sdk/_base/cli/_dorq/_ui/_models.py

```diff
@@ -38,3 +38,19 @@ class TaskRow:
     task_rows: t.Sequence[TaskRow]
     n_tasks_succeeded: int
     n_task_invocations_total: int
+
+
+@dataclass(frozen=True)
+class WFList:
+    """
+    UI model for ``orq wf list``
+    """
+
+    @dataclass(frozen=True)
+    class WFRow:
+        workflow_run_id: str
+        status: str
+        tasks_succeeded: str
+        start_time: str
```

Comment:
**suggestion**: please use structured types like `datetime` in the UI model and leave the human-friendly formatting for the presentation layer `_ui._presenters`